src/Campaigns/Routes/UpdateDefaultCampaignForm.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Give\Campaigns\Routes;
+
+use Exception;
+use Give\Campaigns\Models\Campaign;
+use Give\Campaigns\Repositories\CampaignRepository;
+use Give\Campaigns\ValueObjects\CampaignRoute;
+use WP_REST_Response;
+use WP_REST_Server;
+
+/**
+ * @unreleased
+ */
+class UpdateDefaultCampaignForm
```

Comment:
@glaubersilva I think it will make sense to use the existing `RegisterCampaignRoutes.php` and add this logic there.

Reply:
Resolved in 85524863d9e5e1c925a8b4fda4b6a9c03089d036